app/controllers/admin/attachments_controller.rb

```diff
@@ -103,9 +103,10 @@ def build_file_attachment
 
   def attachment_params
     params.fetch(:attachment, {}).permit(
-      :title, :locale, :isbn, :unique_reference, :command_paper_number,
-      :unnumbered_command_paper, :hoc_paper_number, :unnumbered_hoc_paper,
-      :parliamentary_session, :order_url, :price, :accessible, :external_url,
+      :title, :locale, :isbn, :meta_data_contact_address, :unique_reference,
+      :command_paper_number, :unnumbered_command_paper, :hoc_paper_number,
+      :unnumbered_hoc_paper, :parliamentary_session, :order_url, :price, :accessible,
+      :external_url,
```

Comment:
Totally minor, but as this isn't alphabetical there's no reason to insert the new attr in the middle, so it would be less noisy (and thus clearer what the change was) if you added `:meta_data_contact_address` at the end of the list rather than in the middle.